node/hyperbahn/hyperbahn.thrift

```
@@ -0,0 +1,16 @@
+exception Exception {
```

A catch all exception is useless.